src/repositories/QueueRepository/QueueRepository.ts

```diff
@@ -59,14 +59,16 @@ export class QueueRepository {
    * Removes the BallChaser from the queue with the specified ID
    * @param id Discord ID of the BallChaser to remove from the queue
    */
-  async removeBallChaserFromQueue(id: string): Promise<void> {
+  async removeBallChaserFromQueue(id: string): Promise<ReadonlyArray<Readonly<BallChaser>>> {
     const ballChaserPage = await this.#Client.getById(id);
 
     if (!ballChaserPage) {
       throw new Error(`Cannot remove BallChaser. No BallChaser with the ID ${id} was found.`);
     }
 
     await this.#Client.remove(ballChaserPage.id);
+
+    return this.getAllBallChasersInQueue();
```

Comment:
I don't think we should always return the queue list on remove. This is adding business logic to the repository whose responsibility is supposed to be just performing CRUD operations with the database. If you need the list of ballchasers after removing a player you should make this call in the service. It's possible that we want to remove someone and don't need the updated list which would make this extra async call very wasteful.

Reply:
Good Point.  Removed in next commit.